app/webpack/javascripts/application.js

```diff
@@ -105,7 +105,7 @@ if (!String.prototype.supplant) {
   $('#misc-fees').on('cocoon:after-insert', function (e, insertedItem) {
     const $insertedItem = $(insertedItem)
     moj.Modules.FeeCalculator.UnitPrice.miscFeeTypesSelectChange($insertedItem.find('.fx-misc-fee-calculation'))
-    moj.Modules.FeeCalculator.UnitPrice.feeQuantityChange($insertedItem.find('.js-fee-calculator-quantity'))
+    moj.Modules.FeeCalculator.UnitPrice.miscFeeFormRefresh($insertedItem.find('.js-fee-calculator-quantity'))
```

Comment:
I'm not sure. Aren't both the old and the new lines required here?

Reply:
This cocoon section is specifically targeting the Misc Fees form (#misc-fees), so we only need our new function here. The old function (feeQuantityChange) is used in the fixed fee form on the previous page, elsewhere in code we haven't changed.